zerver/lib/webhooks/common.py

```diff
@@ -166,7 +166,7 @@ def validate_extract_webhook_http_header(
 ) -> Optional[str]:
     assert request.user.is_authenticated
 
-    extracted_header = request.META.get(DJANGO_HTTP_PREFIX + header)
```

Comment:
I think we can also remove `DJANGO_HTTP_PREFIX` entirely?  This was the only use of it.

Reply:
Removed.